install.sh

```diff
@@ -702,6 +702,7 @@ rm -rf /etc/rancher/k3s
 rm -rf /run/k3s
 rm -rf /run/flannel
 rm -rf /var/lib/rancher/k3s
+rm -d /var/lib/rancher
```

Comment:
Could using `rmdir` instead make it easier for the human reader to quickly see what kind of removal this is?